memote-report-app/src/app/test-result.model.ts

```diff
@@ -10,7 +10,7 @@ export class TestResult {
   public format_type: string;
 
   private errorFailsafe(result: string) {
-    if (this.result !== 'skipped' && !this.data ) {
+    if (this.result !== 'skipped' && (this.data == null || typeof this.data === 'undefined') ) {
```

Comment:
Can `this.data` here actually be `'undefined'`? Also, did you use `this.data == null` instead of `this.data === null` on purpose?